loader/src/sources/hyvee/index.js

```diff
@@ -99,7 +101,16 @@ async function fetchRawData() {
     },
   });
 
-  return response.body.data.searchPharmaciesNearPoint;
+  if (response.statusCode >= 400 || response.body.errors) {
+    throw new GraphQlError(response);
+  }
+
+  const result = response.body.data.searchPharmaciesNearPoint;
```

Comment:
I know we didn't have it before, but any benefit to guarding against `response.body` or `response.body.data` being undefined? (If so here, same for `response.body.errors` above)

Reply:
Not `response.body`, but good point that anything inside it should probably be guarded.

Reply:
Should be fixed in f1bdb7afa4e418d4cc3a5dd78e0d308832945d45.